src/orochi-network/chapter.md

```diff
@@ -0,0 +1,49 @@
+<p align="center">
+    <img src="../assets/orochi-network.png" alt="Orochi Network">
+</p>
+
+## What is Orochi Network?
+
+Orochi Network is a project that utilized cryptography, especially Zero-Knowledge Proof to build up an operating system (UnityOS) for the next generation of the internet.
+
+UnityOS manages a pool of computation power. It allows High Performance Decentralized Applications to be hosted and processed without any centralized server. UnityOS runs in a server-less fashion with the underlying technology that can handle almost everything. Hence, the developers only need to focus on designing the logic of their dApps. This layer is built with distributed computing using [zkWASM](https://github.com/orochi-network/zkWASM-specs).
+
+The dApp can be implemented by popular programming languages namely C++, Rust, Go, etc, and processed with semi-native performance. Computation power will be allocated dynamically and thus, saves operational costs and computational waste.
```

Comment:
The dApp can be implemented by popular programming languages, e.g., C++, Rust, Go, etc, and processed with semi-native performance.